tools/Google.Cloud.Tools.ReleaseManager/ContainerCommands/CleanCommand.cs

```diff
@@ -32,33 +31,12 @@ internal class CleanCommand : IContainerCommand
     public int Execute(ContainerOptions options)
     {
         var repoRoot = options.RequireOption(options.RepoRoot);
-        var apiPath = options.ApiPath;
         var rootLayout = RootLayout.ForRepositoryRoot(repoRoot);
         var catalog = ApiCatalog.Load(rootLayout);
 
-        List<ApiMetadata> apis;
-        // No APIs, just clean non-API-specific.
-        if (apiPath == "none")
-        {
-            apis = new();
-        }
-        else if (apiPath is not null)
-        {
-            apis = new List<ApiMetadata>();
-            var targetApi = catalog.Apis.SingleOrDefault(api => api.ProtoPath == apiPath);
-            if (targetApi is null)
-            {
-                Console.WriteLine($"API path '{apiPath}' is not configured for any API. Ignoring.");
-            }
-            else
-            {
-                apis.Add(targetApi);
-            }
-        }
-        else
-        {
-            apis = catalog.Apis.ToList();
-        }
+        // "none" => no APIs, just clean non-API-specific.
+        var apis = options.LibraryId == "none" ? new List<ApiMetadata>()
```

Comment:
Encapsulate this in options? Instead of options.GetApisFromLibraryId(catalog) make it options.GetApis(catalog) and have options know whether it's an empty list, the whole catalog or a specific library (group).

Reply:
Not quite sure I follow - but I'll merge without doing this one *yet*, as I hope to be able to remove "none" anyway, potentially. (It's *only* acceptable for this command.)